src/main/resources/static/payment/checkout.html

```
@@ -1,20 +1,95 @@
 <!DOCTYPE html>
 <html lang="ko">
 <head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>결제위젯 샘플</title>
+  <meta charset="utf-8" />
+  <script src="https://js.tosspayments.com/v2/standard"></script>
 </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)